### PR TITLE
[MM-55107] Replace usage of LocalizedIcon in 'update_user_group_modal.tsx' with i/span tags

### PR DESCRIPTION
**Pull request by akbarkz**
#### Summary

Replace LocalizedIcon with `i` tag in UpdateUserGroupModal component

#### Ticket Link

Fixes https://github.com/mattermost/mattermost/issues/25103
Jira https://mattermost.atlassian.net/browse/MM-55107

#### Screenshots

Screenshot 2023-11-05 at 20 18 49

#### Release Note
```
NONE
```

### CODE REVIEW

File: webapp/channels/src/components/update_user_group_modal/update_user_group_modal.tsx [hunk 1]
```diff
@@ -162,9 +162,9 @@ const UpdateUserGroupModal = (props: Props) => {
                     aria-label='Close'
                     onClick={goBack}
                 >
-                    <LocalizedIcon
+                    <i
                         className='icon icon-arrow-left'
-                        ariaLabel={{id: t('user_groups_modal.goBackLabel'), defaultMessage: 'Back'}}
+                        aria-label={formatMessage({id: 'user_groups_modal.goBackLabel', defaultMessage: 'Back'})}
```
Comment (M-ZubairAhmed) [inline webapp/channels/src/components/update_user_group_modal/update_user_group_modal.tsx:167]: Could you add the ARAI label in the button & remove it from this icon.

**Review by M-ZubairAhmed — CHANGES_REQUESTED** [review]

File: webapp/channels/src/components/update_user_group_modal/update_user_group_modal.tsx (hunk 1 above)
Comment (akbarkz) [reply webapp/channels/src/components/update_user_group_modal/update_user_group_modal.tsx]: done

**Review by M-ZubairAhmed — APPROVED** [review]
Thank you for the work @akbarkz :)

**Comment by M-ZubairAhmed** [comment]
@akbarkz can you please fix the CI

**Review by yasserfaraazkhan — APPROVED** [review]

**Pull request merged**